src/i18n/en.js

```diff
@@ -38,11 +38,11 @@ const messages_en = {
   agencies: {
     breadcrumb: 'Plans',
     breadcrumb_url: `${urlBuyMonthly}`,
-    feature_1: 'Access to the Client Manager panel to manage multiple accounts.',
-    feature_2: 'Administration of user profiles and access permissions for your clients.',
-    feature_3: 'Limitless use of Doppler’s features.',
-    feature_4: 'Advanced Reports with your Campaign’s performances.',
-    feature_5: 'Exclusive Advisory to manage your client’s accounts.',
+    feature_access: 'Access to the Client Manager panel to manage your clients’ accounts. ',
+    feature_admin: 'Administration of user profiles and access permissions for your clients.',
```

Comment:
Shouldn`t we have profile with capital letter also? because it's a doppler entity for Client Manager. Like this:
```suggestion
    feature_admin: 'Administration of user Profiles and access permissions for your clients.',
```
@jmonacchi

Reply:
@cbernat @silsanchez8 In my opinion we should use capital letter only when it's a keyword of the business (like Email Marketing, Marketing Digital, Campaigns, Reports, etc.). I'm not sure if there is any technical limitations or request in that way. So it should be Administration of user profile. Tell me if you see it OK.